src/iop/grain.c

```diff
@@ -496,7 +568,7 @@ void init(dt_iop_module_t *module)
   module->params_size = sizeof(dt_iop_grain_params_t);
   module->gui_data = NULL;
   dt_iop_grain_params_t tmp
-      = (dt_iop_grain_params_t){ DT_GRAIN_CHANNEL_LIGHTNESS, 1600.0 / GRAIN_SCALE_FACTOR, 25.0 };
+      = (dt_iop_grain_params_t){ DT_GRAIN_CHANNEL_LIGHTNESS, 1600.0 / GRAIN_SCALE_FACTOR, 25.0, 0.0 };
```

Comment:
I would guess that the new default behavior should be to produce the grain that makes use of this pr.
I.e. that default should not be `0.0`.